share/completions/lxc.fish

```diff
@@ -0,0 +1,51 @@
+function __fish_lxc_no_subcommand -d 'Test if lxc has yet to be given the command'
+    for i in (commandline --tokenize --cut-at-cursor --current-process)
+        if contains -- $i config copy delete exec file help image launch list move network profile publish remote restore restart snapshot start stop
+            return 1
+        end
+    end
+    return 0
+end
+
+function __fish_lxc_list_containers
+    lxc list -c n | grep '| \w' | sed -e 's/\(|\| \)*//g'
```

Comment:
We prefer using our `string` tool for completions - `string match -r '\| \w+' | string replace -r '\| ' ''`?